lib/renaissance_web/router.ex

```diff
@@ -22,6 +22,8 @@ defmodule RenaissanceWeb.Router do
     get "/login", LoginController, :login
     post "/login", LoginController, :verify
 
+    post "/auctions/:id/update_description", AuctionController, :update_description
```

Comment:
is this accurate? There's no method as far as I can tell?

Reply:
Woops, yes, this is crud, & was resolved with 8ce9b79.